maintainers/maintainer-list.nix

```diff
@@ -9088,6 +9088,12 @@
     githubId = 508305;
     name = "Jaroslavas Pocepko";
   };
+  vonfry = {
```

Comment:
Can the change to `maintainer-list.nix` be in a separate commit titled "maintainers: add vonfry"